Sources/SmithyChecksums/ChunkedStream.swift

```diff
@@ -16,8 +16,8 @@ import class SmithyStreams.BufferedStream
 /// URLSessionHTTPClient streams chunked payloads using this stream type.
 /// CRTClientEngine uses only the reader provided by this type to create chunks, then it
 /// streams them itself.
-public class ChunkedStream {
-    private var inputStream: Stream
+public class ChunkedStream: @unchecked Sendable {
+    private let inputStream: Stream
```

Comment:
Streams were designed to be thread-safe so they may be marked `@unchecked Sendable` so long as we maintain that thread safety.